packages/scaffold-ui/src/views/w3m-connect-view/index.ts

```diff
@@ -76,6 +100,35 @@ export class W3mConnectView extends LitElement {
     return html`<w3m-wallet-login-list></w3m-wallet-login-list>`
   }
 
+  private guideTemplate() {
+    const socials = this.features?.socials
+    const enableWallets = OptionsController.state.enableWallets
+
+    const socialsExist = socials && socials.length
+
+    if (!this.authConnector && !socialsExist) {
+      return null
+    }
```

Comment:
We'll not show the wallet guide UI unless user has enabled email or social login